pom.xml

```diff
@@ -442,6 +446,14 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <repository>
+      <id>central</id>
+      <name>Central Repository</name>
+      <url>https://repo.maven.apache.org/maven2</url>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
```

Comment:
I checked the parent apache pom (v22) to see if it had this repo in it. It doesn't. Just has apache repo.

This looks good. When you clean your local repo, are you downloading from https?

Thanks

Reply:
Yes, verified that.